### ISSUE TITLE
A11y: Some controls are not accessible via keyboard

### ISSUE THREAD

**Issue by ChristofSimon** [grafana/grafana#82172]
**Steps to reproduce**:
1st example: 
manage dashboards, zoom to 200% (with viewport width of 1280 CSS pixels); navigate to menu/selection list via keyboard

2nd example: 
Edit panel, try to operate the control for adjusting the column width without mouse

**Actual Result**:
1st example: 
When the text is enlarged to 200%, the menu becomes a selection list. This cannot be reached and operated via keyboard.
![2 1 1_Keyboard_manage dashbords](https://github.com/grafana/grafana/assets/122528177/22d678d3-4d11-43bf-92e3-42b504dd8cdc)

2nd example: 
The control for adjusting the column width is not accessible by keyboard.
![2 1 1_Keyboard_edit panel](https://github.com/grafana/grafana/assets/122528177/1f135d56-6acd-4055-82d2-ae07f6d16445)

**Expected Result**
All functionality of the content is operable and accessible via keyboard.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
2.1.1 Keyboard: https://www.w3.org/TR/WCAG21/#keyboard

**Environment**:
- Grafana version: 9.1.0
- Data source type & version:
- User OS & Browser: Windows 10, Chrome Version 107.0.5304.88
- Others:

**Comment by ashharrison90** [comment]
both of these are now fixed
